docs/spelling_wordlist

```diff
@@ -472,6 +472,8 @@ Sorani
 sortable
 Spectre
 Springmeyer
+sql
+SQLCompiler
```

Comment:
```suggestion
```
I don't think this is needed

Reply:
The release notes spellcheck was triggering on these.